source/NVDAObjects/UIA/__init__.py

```diff
@@ -1193,10 +1193,14 @@ def _get_UIASelectionPattern(self):
 		return self.UIASelectionPattern
 
 	def _get_UIASelectionPattern2(self):
-		self.UIASelectionPattern2 = self._getUIAPattern(
-			UIAHandler.UIA_SelectionPattern2Id,
-			UIAHandler.IUIAutomationSelectionPattern2
-		)
+		# SelectionPattern2 is not available on older Operating Systems such as Windows 7

```

Comment:
Can you please move this comment into the exception block. So it is more clear that on Windows 7 a COMError is raised.
Same for the other block around line 1469